rebar.config.erl

```diff
@@ -43,7 +50,8 @@ extra_lib_dir() ->
     end.
 
 project_app_dirs() ->
```

Comment:
project_app_dirs is intended to instruct rebar3 finding apps in local dirs to compile
i.e. when not pulled in as rebar dependency.
so I don't think we need to add it here.

Reply:
thank you, removed